js/muchpc.js

```diff
@@ -774,7 +774,7 @@ function defaultRows() {
             Key('1B', 14),
             Key('6', 15, ['0','0','Light']),
             Key('19', 22, ['0','0','Light']),
-            Key('5', 23, ['0', '36']),
+            Key('5', 23, ['0', '34']),
```

Comment:
#6 About this fix, I checked the result.
Maybe committers already know but I share my experience.

I tested on Firmware version 1.04.06.
l checked three cases. this commits results, master-branch and master-branch fixed on gui.   
The each result is below.
|--------------------|---------------|-----------|-----------------|
| type \ version     | commit-marged | master    | master-Fix      |
|--------------------|---------------|-----------|-----------------|
| "b + fn" result    | < (wrong)     | < (wrong) | " (correct)     |
|--------------------|---------------|-----------|-----------------|
| web_site_preview   | " (correct)   | < (wrong) | " (fix by hand) |
|--------------------|---------------|-----------|-----------------|
| preview_reflection | wrong         | correct   | correct         |
|--------------------|---------------|-----------|-----------------|

As a result, this commits fix the default key-map but binary making process go wrong.
The web site on master branch's default key map is wrong but user can fix on GUI and it works properly.
I think the preview reflection error(binary-making error) is bigger issue for users than default key-map error.
This commits shouldn't be merged.

I wish the default key-map error fixed. But it is complected to fix for me.
So I share test result details.  https://github.com/anosillus/much-programming-core/commit/2b79bc4e6094795f0ec38296762beeefa9f524db